src/main/java/io/voucherify/client/model/loyalties/AddBalance.java

```diff
@@ -15,4 +15,6 @@
 public class AddBalance {
 
   private Long points;
+  private String expiration_date;
+  private String expiration_type;
```

Comment:
Please rename variables to camelCase notation and add @JsonProperty annotations. Make enum for `expirationType` with elements `PROGRAM_RULES, CUSTOM_DATE, NON_EXPIRING` according to https://docs.voucherify.io/reference/add-loyalty-card-balance